tests/end_to_end/models/participants.py

```diff
@@ -132,13 +133,13 @@ def modify_plan(self, new_rounds=None, num_collaborators=None):
 
         data["aggregator"]["settings"]["rounds_to_train"] = int(self.rounds_to_train)
         data["data_loader"]["settings"]["collaborator_count"] = int(self.num_collaborators)
+        data["network"]["settings"]["disable_client_auth"] = True if disable_client_auth else False
+        data["network"]["settings"]["tls"] = False if disable_tls else True
```

Comment:
similarly, here we could simply write `data["network"]["settings"]["tls"] = not disable_tls` ?

Reply:
Done.